nixos/tests/installed-tests/xdg-desktop-portal.nix

```diff
@@ -1,33 +1,34 @@
-{ pkgs, makeInstalledTest, ... }:
+{
+  lib,
+  pkgs,
+  makeInstalledTest,
+  ...
+}:
 
 makeInstalledTest {
   tested = pkgs.xdg-desktop-portal;
 
-  # Red herring
-  # Failed to load RealtimeKit property: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.RealtimeKit1 was not provided by any .service files
-  # Maybe a red herring, enabling PipeWire doesn't fix the location test.
-  # Failed connect to PipeWire: Couldn't connect to PipeWire
   testConfig = {
     environment.variables = {
+      GI_TYPELIB_PATH = lib.makeSearchPath "lib/girepository-1.0" [
+        pkgs.glib.out
+        pkgs.umockdev.out
+      ];
+      # need to set this ourselves, because the tests will set LD_PRELOAD=libumockdev-preload.so,
+      # which can't be found because it's not in default rpath
+      LD_PRELOAD = "${pkgs.umockdev.out}/lib/libumockdev-preload.so";
       XDP_TEST_IN_CI = 1;
       XDG_DATA_DIRS = "${pkgs.xdg-desktop-portal.installedTests}/share/installed-tests/xdg-desktop-portal/share";
     };
     environment.systemPackages = with pkgs; [
-      (python3.withPackages (
-        pp: with pp; [
-          pygobject3
-          sphinxext-opengraph
-          sphinx-copybutton
-          furo
-        ]
-      ))
+      umockdev
       wireless-regdb
     ];
     # Broken, see comment in the package file.
```

Comment:
If the location test works now, this is no longer relevant.

Reply:
Oops, forgot to remove this.